CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -837,6 +837,16 @@ function PlayerHospital:afterLoad(old, new)
   if old < 159 then
     self.adviser_data.reception_advice = self.adviser_data.reception_advice or self.receptionist_msg
   end
+  if old < 210 then
+    -- Serious Radiation epidemics could exist in old saves and cause a crash
+    local serious_radiation = TheApp.diseases["serious_radiation"]
+    if self.epidemic and self.epidemic.disease == serious_radiation then
+      -- Tell the player somehow we ended the epidemic, EPI0008 is most fitting
+      self.world.ui:playAnnouncement("EPID008.wav", AnnouncementPriority.Critical)
+      self.world:gameLog("Notice: Removing active epidemic for Serious Radiation. See issue 2760.")
```

Comment:
```suggestion
      self.world:gameLog("Notice: Removing active epidemic for Serious Radiation as it may crash the game due to a bug.")
```

Reply:
Users probably don't even know what issue 2760 is.

Reply:
This is probably more useful for the spd_log draft concept I agree. Though gamelog *is useful to us*.
Nevertheless I'll drop it.

Reply:
You could mention the issue as a side-remark, it just shouldn't be the explanation itself.

For us, the usual way in is to search that phrase in git and from the revision with the change you can then find what was done and what issues were involved.

Another option is to add the issue number as comment to the code there.  
The problem of that is aging.